materialrangebar/src/main/java/com/appyvet/materialrangebar/RangeBar.java

```diff
@@ -920,6 +969,44 @@ public void setPinRadius(float pinRadius) {
         createPins();
     }
 
+    /**
+     * Sets left selector circle color
+     *
+     * @param mCircleColorLeft
+     */
+    public void setSelectorColorLeft(int mCircleColorLeft) {
```

Comment:
`setSelectorColorLeft` should be renamed to `setLeftSelectorColor` to be consistent with the xml properties